docs/src/main/asciidoc/cdi-integration.adoc

```diff
@@ -169,20 +169,20 @@ With an `AnnotationsTransformerBuildItem` it's possible to override the annotati
 NOTE: Keep in mind that annotation transformers must be produced _before_ the bean discovery starts.
 
 For example, you might want to add an interceptor binding to a specific bean class.
-You can use a convenient builder-like API to create a transformer instance:
+You can use a convenient builder API to create a transformation instance:
 
-Builder Example
+.Builder Example
 [source,java]
 ----
 @BuildStep
 AnnotationsTransformerBuildItem transform() {
-   return new AnnotationsTransformerBuildItem(AnnotationsTransformer.appliedToClass() <1>
-        .whenClass(c -> c.name().toString().equals("org.acme.Bar")) <2>
-        .thenTransform(t -> t.add(MyInterceptorBinding.class))); <3>
+    return new AnnotationsTransformerBuildItem(AnnotationTransformation.forClasses() // <1>
+        .whenClass(DotName.createSimple("org.acme.Bar")) // <2>
+        .transform(t -> t.add(MyInterceptorBinding.class))); // <3>
```

Comment:
For the future, IIRC, you don't need the `//`. The callouts are automatically removed when you copy/paste the code.
It doesn't hurt though.

Reply:
I know they are not necessary, but they help IDEA understand the code. That's why I've been using them everywhere :-)

Reply:
OK, makes sense then!